fe/fe-core/src/main/java/org/apache/doris/nereids/parser/ParseDialect.java

```diff
@@ -0,0 +1,111 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.nereids.parser;
+
+/**
+ * ParseDialect enum, maybe support other dialect.
+ */
+public enum ParseDialect {
+
+    /**
+     * Trino parser and it's version is 395.
+     */
+    TRINO_395(Dialect.TRINO, Version.TRINO_395),
+    /**
+     * Doris parser and it's version is 2.0.0.
+     */
+    DORIS_2_(Dialect.DORIS, Version.DORIS_2_);
```

Comment:
why suffix `_` exist?

Reply:
The version `DORIS_2_` represent all versions for Doris 2 in the future，such as 2.0, 2.1, 2.22，maybe `Version.DORIS_2_*` is better, WDYT?